csrankings-0.csv

```diff
@@ -1722,6 +1722,7 @@ Armin R. Mikler,University of North Texas,http://www.cse.unt.edu/~mikler/A.R.Mik
 Armin Zimmermann,TU Ilmenau,https://www.tu-ilmenau.de/sse/mitarbeiter/prof-zimmermann,HkSd-OMAAAAJ
 Armon Barton,Naval Postgraduate School,https://armonbarton.com,ug8vrqQAAAAJ
 Arnab Bhattacharyya 0001,National University of Singapore,https://www.comp.nus.edu.sg/~arnab,eECXWqUAAAAJ
+Arnab Bhattacharya  0001, IIT Kanpur,https://www.cse.iitk.ac.in/users/arnabb/ 

```

Comment:
* One space between name and "0001"
* No spaces after commas
* Missing Google Scholar ID